docs/proposal-rose-suite-run.md

```diff
@@ -136,16 +231,28 @@ sources on installation. Other things to consider:
 The `rose suite-run` command calls `cylc validate --strict` by default.
 Automatic suite validation should become the default behaviour for `cylc run`
 and `cylc reload`. Things to consider:
-* Is the *strict* mode necessary by default?
-* `rose suite-run` does not currently validate Rose items. This should be
-  changed to ensure that all aspects of the suite are validated uniformly.
+* `rose suite-run` does not currently validate Rose apps against their
+  metadata. Rose apps should be validated against their metadata by default,
+  but contain switches in the manner of bandit or pylint in a config file.
+  (perhaps `cylc-validate.yml` or similar)
+* Creators of largers suites may want to turn this because the validation
```

Comment:
may want to turn this on or off?

Reply:
I've actuall removed this after a conversation with Oliver.